pkg/async/notifications/implementations/sandbox_processor_test.go

```diff
@@ -0,0 +1,39 @@
+package implementations
+
+import (
+	"context"
+	"testing"
+
+	"github.com/flyteorg/flyteadmin/pkg/async/notifications/mocks"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+)
+
+var mockSandboxEmailer mocks.MockEmailer
+
+func TestSandboxProcessor_UnmarshalMessage(t *testing.T) {
+	var emailMessage admin.EmailMessage
```

Comment:
What's the purpose of this test? Are we just verifying we can unmarshal a proto? Considering that test code already initializes this msg var I'm not sure how valuable this unit test is.

Reply:
Yes, you are right, I can consider deleting it.